tests/integration-test/bpf/build_push_images.sh

```diff
@@ -24,7 +24,7 @@ docker push quay.io/bpfd-bytecode/tc_pass
 
 docker build \
  --build-arg PROGRAM_NAME=tracepoint \
- --build-arg SECTION_NAME=sys_enter_openat \
+ --build-arg SECTION_NAME=enter_openat \
```

Comment:
Doesn't this need to match the section name defined in the following?
https://github.com/bpfd-dev/bpfd/blob/main/tests/integration-test/bpf/tp_openat.bpf.c#L15

If so, one or the other will need to change and get rebuilt and pushed.

Reply:
Nope with https://github.com/aya-rs/aya/pull/413 it's now based on the actual function name 

( I already built and pushed the integration test images)